src/roi/Roi.ts

```diff
@@ -495,7 +527,12 @@ function getInternalIDs(roi: Roi) {
 
   return internal;
 }
-
+/**
+ * Not sure what it does
+ *
+ * @param roi
+ * @returns
+ */
```

Comment:
Avoid adding personal comments to the the code

You don't need to add comment to every function.